### PR TITLE
feat: 1671 extend config to have all sdk standard components endpoints

### PR DESCRIPTION
**Pull request by eoln**
- also fix braking change for v12 in sdk-standard-components

### CODE REVIEW

**Comment by lewisdaly** [comment]
Hmm tests are failing @eoln

File: src/shared/config.ts [hunk 1]
```diff
@@ -77,48 +78,19 @@ export interface ServiceConfig {
...
     PEER_ENDPOINT: string
-    ALS_ENDPOINT: string
-    QUOTES_ENDPOINT: string
-    TRANSFERS_ENDPOINT: string
-    BULK_TRANSFERS_ENDPOINT: string
+    ALS_ENDPOINT?: string
+    QUOTES_ENDPOINT?: string
+    TRANSFERS_ENDPOINT?: string
+    BULK_TRANSFERS_ENDPOINT?: string
+    THIRDPARTY_REQUESTS_ENDPOINT?: string
```
Comment (sridharvoruganti) [inline src/shared/config.ts:85]: It seems to be forgot to add `THIRDPARTY_REQUESTS_ENDPOINT` and `TRANSACTION_REQUEST_ENDPOINT` in config files

**Review by kleyow — COMMENTED** [review]
lgtm after sridhar's comment is addressed

File: src/shared/config.ts (hunk 1 above)
Comment (eoln) [reply src/shared/config.ts:85]: there are optional parameters, if not specified PEER_ENDPOINT will be used

**Review by sridharvoruganti — APPROVED** [review]

**Pull request merged**